src/promptflow/promptflow/executor/assistant_tool_invoker.py

```diff
@@ -0,0 +1,97 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
```

Comment:
We may not expose this as a public class since we are not ready yet.

Reply:
Changed to internal.